docs/source/tutorials/object-detector.md

```diff
@@ -1 +0,0 @@
-../../../label_studio/ml/examples/mmdetection/README.md
```

Comment:
These tutorials are outdated - need to sync with what we have in label-studio-ml-backend/examples